lib-multisrc/mangahub/src/eu/kanade/tachiyomi/multisrc/mangahub/MangaHubDto.kt

```diff
@@ -0,0 +1,84 @@
+package eu.kanade.tachiyomi.multisrc.mangahub
+
+import kotlinx.serialization.Serializable
+
+typealias ApiChapterPagesResponse = ApiResponse<ApiChapterData>
+typealias ApiSearchResponse = ApiResponse<ApiSearchObject>
+typealias ApiMangaDetailsResponse = ApiResponse<ApiMangaObject>
+
+// Base classes
+@Serializable
+class ApiResponse<T>(
+    val data: T?,
+    val errors: List<ApiErrorMessages>?,
+)
+
+@Serializable
+class ApiErrorMessages(
+    val message: String,
+)
+
+// Chapter metadata (pages)
+@Serializable
+class ApiChapterData(
+    val chapter: ApiChapter?,
```

Comment:
here too(?)

Reply:
Context (resolved): https://github.com/keiyoushi/extensions-source/pull/8392#discussion_r2037770535
> don't make nullable since you are checking for errors in interceptor